java/src/jmri/jmrix/zimo/Mx1SystemConnectionMemo.java

```diff
@@ -141,11 +141,6 @@ public boolean provides(Class<?> type) {
      * hexfile.HexFileFrame and locormi.LnMessageClient
      */
     public void configureManagers() {
-        
-        /* Already created during config with default settings. Recreating
-        * now that the traffic controller exists
-        */
-        programmerManager = null;
```

Comment:
@pabender The above fix eliminates the need to reset the program manager to null and it creates the program manager with the traffic controller.  I have tested the fix and I am able to read /write my decoders again. 👍